weapon-config.inc

```diff
@@ -1926,133 +1930,133 @@ stock PlayerText:WC_CreatePlayerTextDraw(playerid, Float:x, Float:y, text[])
 stock WC_PlayerTextDrawDestroy(playerid, PlayerText:text)
 {
 	if (playerid < 0 || playerid >= MAX_PLAYERS) return 0;
-	if (_:text < 0 || text >= MAX_PLAYER_TEXT_DRAWS || s_InternalPlayerTextDraw[playerid][text]) return 0;
+	if (_:text < 0 || _:text >= MAX_PLAYER_TEXT_DRAWS || s_InternalPlayerTextDraw[playerid][text]) return 0;
```

Comment:
A better solution for this would be:

```pawn
if (_:text < 0 || text >= PlayerText:MAX_PLAYER_TEXT_DRAWS || s_InternalPlayerTextDraw[playerid][text]) return 0;
```

I don't know why the symbol doesn't have a tag anyway, but you are best to apply it there, so the code works if the tag exists or doesn't.

Reply:
I am not sure about tag, he give me a warning on this line for text why, i tried with that but for me this workin ok

Reply:
Yes, that will work in some cases, but not all.  The version I posted covers all cases.

Reply:
Okay, i will try to help improve this :D